docs/index.md

```diff
@@ -1,3 +1,14 @@
 # PrOMMiS Documentation
 
 Welcome to the PrOMMiS documentation!
```

Comment:
PrOMMiS (Process Optimization and Modeling for Minerals Sustainability) is a multi-institutional collaboration building a software modeling platform to accelerate development and deployment of extraction and purification processes for critical minerals and rare earth elements at reduced risk.

PrOMMiS helps to guarantee national strategic objectives for critical materials:
• Secure supply chains
• Economic competitiveness
• Energy security and independence

PrOMMiS leverages tools and models from the Institute
for Design of Advanced Energy Systems (IDAES), Carbon Capture Simulation for Industry Impact (CCSI2) and other related programs.